src/style/utils/filter-out-styled-system-spacing-props.ts

```diff
@@ -1,7 +1,8 @@
 import styledSystemPropTypes from "@styled-system/prop-types";
 
 const filterOutStyledSystemSpacingProps = (
-  obj: Record<string, unknown>
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  obj: Record<string, any>
```

Comment:
I think that we should always add a comment in the code explaining why is a rule disabled,
I would also change the name `obj` to `props`
eg.:

```suggestion
  // method should accept any react prop
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  props: Record<string, any>
```